File: Data Structures & Algorithms/two-integer-sum-ii/submission-1.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int ans[]=new int[2];
        Map<Integer,Integer> mp=new HashMap<>();
        for(int i=0;i<numbers.length;i++){
            mp.put(numbers[i],i);
        }
        for(int i=0;i<numbers.length;i++){
            int diff=target-numbers[i];
            if(mp.containsKey(diff)){
                ans[0]=mp.get(diff)+1;
                ans[1]=i+1;
            }
        }
        return ans;
    }
}

File: Data Structures & Algorithms/is-subsequence/submission-1.py
class Solution:
    def isSubsequence(self, s: str, t: str) -> bool:
        if len(s)>len(t): return False
        mp={}

        for i in s:
            mp[i]=mp.get(i,0)+1
        
        for j in t:
            if j in mp:
                mp[j]=mp.get(j,0)-1

        print(mp)
        for val in mp.values():
            if val>0:
                return False

        return True

                
        